x = 5

try:
  print(x)
except:
  print("Ocurri√≥ un error")